packages/flutter_tools/bin/fuchsia_builder.dart

```diff
@@ -37,6 +38,7 @@ Future<Null> main(List<String> args) async {
   executableContext.setVariable(Logger, new StdoutLogger());
   executableContext.runInZone(() {
     // Initialize the context with some defaults.
+    context.putIfAbsent(FileSystem, () => new LocalFileSystem());
```

Comment:
I really would prefer dropping the use of zones here in favour of explicitly passing in a "context" variable of some sort, but that's a story for another day.